triage/2023-03-28.md

```diff
@@ -0,0 +1,234 @@
+# 2023-03-28 Triage Log
+
+A busy week with lots of real performance gains. Most regressions seemed to be due to noise. The biggest highlight was large wins in incremental compilation leading to a lot of (albeit modest gains) of 1% performance in a majority of incremental compilation test scenarios. Other than that most performance gains were smaller and more incremental. One of the biggest performance regressions came in an update too LLVM. However, nearly just as many test cases showed improvements as regressions.
```

Comment:
```suggestion
A busy week with lots of real performance gains. Most regressions seemed to be due to noise. The biggest highlight was large wins in incremental compilation leading to a lot of (albeit modest) gains of 1% performance in a majority of incremental compilation test scenarios. Other than that most performance gains were smaller and more incremental. One of the biggest performance regressions came in an update to LLVM. However, nearly just as many test cases showed improvements as regressions.
```

Reply:
"more incremental" -- is that referring to incremental compilation, or just the sense of "minor"? If the latter, maybe change it to avoid potential confusion.

Reply:
Also, for the LLVM upgrade, I [commented](https://github.com/rust-lang/rust/pull/109474#issuecomment-1487857994) in the PR about how the cycle and wall-time results of that change look much better than the instruction count results.